4-architecture.md

```diff
@@ -67,7 +67,7 @@ Protocol multiplexing is done through [`multistream`](https://github.com/jbenet/
 
 ### 4.2.5 Identify
 
-**Identify** is one of the protocols mounted on top of Swarm, our Connection handler. However, it follows and respects the same pattern as any other protocol when it comes to mounting it on top of Swarm. Identify enables us to trade `listenAddrs` and `observedAddrs` between peers, which is crucial for the working of IPFS. Since every socket open implements `REUSEPORT`, an `observedAddr` by another peer can enable a third peer to connect to us, since the port will be already open and redirect to us on a NAT.
+**Identify** is one of the protocols mounted on top of Swarm, our Connection handler. However, it follows and respects the same pattern as any other protocol when it comes to mounting it on top of Swarm. Identify enables us to trade listened addresses and observed addresses between peers, which is crucial for the working of libp2p. Since every socket open implements `REUSEPORT`, an address observed by another peer can enable a third peer to connect to us, since the port will be already open and redirect to us on a NAT.
 
```

Comment:
This is technically inaccurate. The observed address allows us to infer the presence of a NAT and port stability. This _may_ allow us to traverse some NATs (full-cone tcp).